### ISSUE TITLE
Opening a file from SharePoint's "Open in Explorer" window causes lots of Ethernet traffic and cursor loading flickers

### ISSUE THREAD

**Issue by SokhaS** [Microsoft/vscode#43661]
- VSCode Version: 1.20.0
- OS Version: Windows 10 Version 1709 Build 16299.192

Steps to Reproduce:

1. Using SharePoint 2013, navigate to a document library where a .js file is stored.
2. Click "Open in Explorer"
3. Drag the file into VSCODE
4. Notice that the cursor constantly flickers the loading symbol and constant spike in Ethernet traffic.

This does not happen in version 1.19. I assumed it was the new error showing in explorer update, but disabling that doesn't resolve the issue. I believe it's something new that's constantly polling the file for information.

Does this issue occur when all extensions are disabled?: Yes!

**Comment by isidorn** [comment]
Does this also happen if you open that file in some other editor, like atom sublime visual studio?
Can you also reproduce with vscdoe insiders?

**Comment by SokhaS** [comment]
Using the same .js file I tested in a few editors.
**VS Insider** : Yes it still happens
**VS Pro 2015**: No
**Atom**: No
**Sublime Text 3**: No
**Notepad**: No

**Comment by SokhaS** [comment]
Additional information:
I believe SharePoint's "Open in Explorer" mode is using WebDAV. Maybe that will help. 
It **does not happen** when the file is local on my machine.

**Comment by isidorn** [comment]
@bpasero any hints on this one?

**Comment by SokhaS** [comment]
I have also tested with different file extensions. It doesn't do it with .css or .html.
It seems to only do it on .js files.
I have confirmed that it happens on older versions as well. 1.19.3, 1.18, and 1.17. Which means it might not have to do with something new.

**Comment by bpasero** [comment]
Can you try to reproduce with our nightly insider builds? You can give our preview releases a try from: https://code.visualstudio.com/insiders/

**Comment by SokhaS** [comment]
Yes, the issue still occurs in the insider version. However, it doesn't happen in VS Pro 2015, Sublime, or Atom.
